translations/kubectl/pl_PL/LC_MESSAGES/pl_PL.po

```diff
@@ -0,0 +1,44 @@
+# Test translations for unit tests.
+# Copyright (C) 2016
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR brendan.d.burns@gmail.com, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: kubectl\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2017-01-27 13:57+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.11\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n"
+"%100<10 || n%100>=20) ? 1 : 2);\n"
+"Last-Translator: \n"
+"Language: pl_PL\n"
+
+msgctxt "Update the annotations on a resource"
+msgid "Update the annotations on a resource"
+msgstr "Zaktualizuj adnotacje dla zasobu"
```

Comment:
Yeah, but here the annotation is on a resource, not for. For suggests being something separate, like normal annotation in a book, not on a object. I was also considering word "anotacja" instead of adnotacja to make the difference cleaner, but I'm not convinced :/